app/models/concerns/relatable.rb

```diff
@@ -28,6 +28,8 @@ def self.get_related_model(counter_name)
       counter_name = counter_name.split(/ as /i)[-1]  # honor aliasing
       counter_name.gsub!('_count','')   # support cached count columns as well
 
+      counter_name.gsub!('qtls','qtl')  # unfortunate special case... :(
```

Comment:
Guys, thanks for having this long, academic, rhetoric, opinionated, colorful, firebrand discussion about how to add me a lot of work. I would much prefer to have that kind of discussions taking place, like, 03.03, instead of two months later :).

Still, if Piotr wants to change table, model and has_many relations names from qtl to qtls, please be it rather sooner than later!